frontend/packages/helm-plugin/src/components/forms/HelmChartRepository/helmchartrepository-validation-utils.ts

```diff
@@ -27,7 +27,20 @@ export const createHelmChartRepositoryValidationSchema = (t: TFunction) =>
         message: t('helm-plugin~Invalid Repo URL.'),
       })
       .max(2048, t('helm-plugin~Please enter a URL that is less then 2048 characters.'))
-      .required(t('helm-plugin~Required')),
+      .required(t('helm-plugin~Required'))
+      .test(
+        'https-with-basic-auth',
+        t(
+          'helm-plugin~Basic authentication requires HTTPS. Using HTTP with credentials is insecure.',
+        ),
```

Comment:
Just out of curiosity, why do we use such a long key for the translation?  That is, the key doesn't _have_ to be the same as the English translation, right?...so why not use something shorter and ease the impact on the source code?
```suggestion
        t('helm-plugin~Basic authentication requires HTTPS'),
```
The translation can still include the comment about lack of security, it just doesn't have to be included _here_.